containerimages/container-disk-images/create-push-vm-image.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+set -ex
+
+SCRIPT_DIR="$(
+    cd "$(dirname "$BASH_SOURCE[0]")"
+    pwd
+)"
+
+export REGISTRY=${REGISTRY:-quay.io}
+export REPOSITORY=${REPOSITORY:-omergi}
+export IMAGE_NAME=${IMAGE_NAME:-fedora-extended}
+export TAG=${TAG:-tests}
+
+readonly VM_IMAGE_URL="https://download.fedoraproject.org/pub/fedora/linux/releases/32/Cloud/x86_64/images/Fedora-Cloud-Base-32-1.6.x86_64.qcow2"
+readonly VM_IMAGE="source-image.qcow2"
+
+full_image_tag="$REGISTRY/$REPOSITORY/$IMAGE_NAME/$TAG"
+build_directory="${IMAGE_NAME}_build"
+new_vm_image_name="provisioned-image.qcow2"
+
+trap 'cleanup' EXIT
+
+function cleanup() {
+    echo "cleanup"
+    rm -rf $build_directory || true
```

Comment:
u dont need || true because there is -f

Reply:
I will change it in following PR at kubevirtci as discussed in last community meeting.